js/src/components/weeklygreen.tsx

```diff
@@ -0,0 +1,48 @@
+import React from "react";
+import { Col, Row, Tooltip } from "antd";
+import { SiteWeeklyGreenMetric } from "../interface";
+import "./segment.css";
+
+interface Prop {
+  data: Array<SiteWeeklyGreenMetric>;
+}
+
+const WeeklyGreenMetric: React.FC<Prop> = ({ data }) => {
+  return (
+    <>
```

Comment:
actually never knew this `<>` trick.

do you still need the `(` before `<>`?

Reply:
i just copy from other components; and yes syntax error without brackets

Reply:
I thought it can accept

````
return <>
   ...
</>
````

but I might be wrong.

anyways, good to be consistent.